### DISCUSSION TITLE
Problem with applying for GH Education

### DISCUSSION THREAD

**Discussion by DavideDunne** [community/community#65491] [GitHub Education]
### Hi everyone,

Hi, I got an account with my college email back in my first year. It has been abandoned since then and therefore I don't have any activity in it, however my main account with my personal email has been more active with OSS contributions and personal/school projects.

Should I delete the account with my college email and then link my college email with my active account in the [Github Education application](https://education.github.com/discount_requests/application)? Should that work?

Thanks

**Comment by unknown4now** [comment]
> ### Hi everyone,
> Hi, I got an account with my college email back in my first year. It has been abandoned since then and therefore I don't have any activity in it, however my main account with my personal email has been more active with OSS contributions and personal/school projects.
> 
> Should I delete the account with my college email and then link my college email with my active account in the [Github Education application](https://education.github.com/discount_requests/application)? Should that work?
> 
> Thanks
> 
> ### A suggestion...(optional)
> _No response_

Hello.Yes, you can definitely consider deleting the GitHub account associated with your college email address if it's no longer in use or has been abandoned. But make sure to back up any important data or repositories from that account if needed.
You can then use your college email address to link it with your active GitHub account through the GitHub Education application, if you're eligible for the GitHub Education benefits. This should work, as long as you meet the eligibility criteria for the GitHub Education benefits using your college email.

**Comment by Gihansachith92** [comment]
Hi.If your college email address is no longer in use or has been abandoned, you should absolutely think about canceling the GitHub account that is connected to it. But if necessary, make sure to backup any crucial data or repository locations from that account.
If you qualify for the GitHub Education advantages, you can then use your college email address to connect it with your active GitHub account using the GitHub Education application. As long as you are eligible for the GitHub Education benefits using your college email, this should work.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
